desktop/core/src/desktop/js/reactComponents/FileChooserComponent/FileChooserComponentButton/FileChooserComponentButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+import './FileChooserComponentButton.scss';
+import FileChooserModal from '../FileChooserModal/FileChooserModal';
+import { Button } from 'antd';
```

Comment:
nit: We can group the import statements with external libraries first and then our internal imports